src/App.tsx

```diff
@@ -963,7 +981,7 @@ function App(props: RouteComponentProps) {
                         {!isMinimalMode && (
                             <div
                                 style={{
-                                    width: '100%',
+                                    width: `calc(100% - ${0}px)`,
```

Comment:
I assume this change was only for testing purposes?

```suggestion
                                    width: '100%',
```